packages/jss-plugin-extend/src/index.d.ts

```diff
@@ -1,3 +1,7 @@
 import {Plugin} from 'jss'
 
+export type ExtendProperties = {
+  extends: string[]
```

Comment:
- it's "extend"
- it's much more complex than that, please check the docs https://cssinjs.org/jss-plugin-extend?v=v10.5.1